rules/_deprecated/credential_access_symbolic_link_to_shadow_copy_created.toml

```diff
@@ -1,7 +1,8 @@
 [metadata]
 creation_date = "2021/12/25"
-maturity = "production"
-updated_date = "2022/02/16"
+deprecation_date = "2022/03/10"
+maturity = "deprecated"
```

Comment:
@terrancedejesus If I remember correctly, the copy command, for example, does not generate a process creation event or Copy-Item (Unless there is a `cmd/powershell /C command`, so there are limitations on what the other rule can detect.
I don't see a case to deprecate this, as it can be the only trace to the activity in some scenarios. Maybe we can discuss a severity change to low... wdyt @Samirbous?

Reply:
Tbh, this one can be weak on that point too, as mklink does not generate a process creation event too as far as I can remember

Reply:
Yes in order to recreate I had to load powershell first, add the command as a link and point to cmd so there was a process creation event. Can confirm on copy as well. Will explore this a bit more thanks for the insight!

Reply:
Would be curious to know if I am missing something now.

When I run the commands `copy` or `mklink` while in CLI, that data does not appear in my cluster based on `process.args` or `process.command_line`. Why could that be? My agent is has a policy that has endpoint, windows, system and pre-built rules as integrations and the host I am testing on also has Sysmon setup.

If I run a command via POSH prompt and pipe it to CMD as shown below, I see the parent-child process relationship but not the args or command line data. 

Steps to recreate:

1. Create a restore point (shadow copy) for C:\ Drive
2. Verify shadow copy by opening cmd.exe as admin and running `vssadmin List Shadows` to find location of shadow copy
3. If no AD, create a blank file called ntds.dit at `C:\Windows\NTDS\ntds.dit`.
4. Start PS as admin, then execute `"mklink /d C:\vsscopy \?\GLOBALROOT\Device\HarddiskShadowVolumeCopy1\"  | cmd` to create a symbolic link
5. In PS, execute `"copy \\?\GLOBALROOT\Device\HarddiskVolumeShadowCopy1\Windows\NTDS\ntds.dit C:\ntdscopy.dit" | cmd`

From here we would copy the boot key from the registry, which is detected by endpoint rules. My confusion is that I am unable to find the actions taken above in discover as if the data does not exist. Maybe I have my setup wrong??

Reply:
>the copy command, for example, does not generate a process creation

Yeah, this is the behavior. As I stated, this does not generate an event because no process is created when using mklink or copy. Process creations happen (But not only) when we run commands that are associated with system binaries; take this as an example:

![image](https://user-images.githubusercontent.com/26856693/157759921-d7747ef1-db97-4e5d-aa14-dc80c4d82788.png)

Reply:
This is the point that I said that makes this kind of rules kinda weak, the attacker needs to execute another binary (Cmd/Powershell) to execute the command so we can have visibility on what was run. For example:

PS C:\Users\Administrator> cmd.exe /C "mklink /d C:\vsscopy \?\GLOBALROOT\Device\HarddiskShadowVolumeCopy1\"

Reply:
Very good point, thank you for covering this. I misunderstood the necessity to spawn a brand new process in order for us to have visibility and have now confirmed I can see the data. Looking forward to thoughts from Samir as well.

Reply:
Took some time to dig deeper into the intrusion effort and offer a sequence that would eliminate potential FPs instead of just deprecating the rule after a conversation with @w0rk3r. Added possibility of PowerShell for symbolic link creation and sequenced with directly copying the ntds.dit file via CMD or PowerShell. 

<img width="1912" alt="Screen Shot 2022-03-14 at 1 04 39 PM" src="https://user-images.githubusercontent.com/99630311/158223529-7962999e-8cf0-4469-813f-7bf8e1ad5f45.png">


Ready for review of the rule and any other potential tuning. Thanks!

Reply:
built-in commands within shells do not trigger new process events, only if they are calls to existing binaries (which is why `mklink` or `copy` do not show up)